app/models/user.rb

```diff
@@ -125,7 +126,11 @@ def with_all_associations_joined
     def with_interest(interest)
       where(":interest = ANY(interested_in)", interest: interest)
     end
-  end
+
+    def immutable_params
```

Comment:
:sob: true, it was suppose to be attributes, sorry for that

indentation - `def immutable_attributes` is in class << self block